recipes/numpoly/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.1.8" %}
+
+package:
+  name: numpoly
+  version: {{ version }}
+
+source:
+  fn: numpoly-{{ version }}.tar.gz
+  url: https://github.com/jonathf/numpoly/archive/v{{ version }}.tar.gz
+  sha256: 1c8a7b1343abbe0f407e1f0cbdfbd0270cfab8eef059d9a45f985b076f854534
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - python >=3.5
```

Comment:
In general, only `pip` and `python` are needed in `host`, and the other dependencies should go in `run`. Also, I don't think there is need of including `setuptools`.

Reply:
ok